resolvers.js

```diff
@@ -67,7 +67,15 @@ const resolvers = {
 
       return questionnaireResponseItem.answer;
     }
-  }
-};
+	  //args: id of the patient
+	  //Using the patient id, find the correspoding patient
+    patient(root, args) {
+      return Fhir.getOne({ resource: 'Patient', id: args.id }).then(res => {
+	    console.log(res)
+	    const result = res.entry.map(entry => entry.resource)
+	    return result;
+
+      }
+    };
```

Comment:
The linter is showing some syntax errors, looks like this should be a big object.

<img width="580" alt="screen shot 2019-01-18 at 11 43 32 pm" src="https://user-images.githubusercontent.com/24414632/51422337-ef85e380-1b7a-11e9-951b-ddc06c63f16f.png">